Custom prefix per-reference (#961)
quarto-dev/quarto-web: docs/books/book-crossrefs.qmd

See @sec-introduction for additional discussion.
```

> To suppress cross-reference prefix prepend `-` before `@`, like so `[-@citation]`, will suppress default prefix, e.g. `[-@sec-visualization]` will produce just `1.1` (or whatever number this section happens to be) without `Section` prefix.

Reviewer: Maybe we should mention bracket syntax to be completely exact ... 🤔 

```suggestion
To suppress cross-reference prefix, use bracket syntax and prepend `-` before `@`, like so `[-@citation]`. This will suppress default prefix, e.g. `[-@sec-visualization]` will produce just `1.1` (or whatever number this section happens to be) without default prefix.
```

``` markdown
See [-@sec-visualization].
```

> To refer to a chapter or appendix explicitly you should spell out "Chapter" or "Appendix" and use the number-only form of cross reference:

Reviewer: If the number-only form is unneeded, mention should be removed. 

```suggestion
To refer to a chapter or appendix explicitly you should spell out "Chapter" or "Appendix" as prefix in the bracket syntax for cross reference:
```

Right ?
